observability/otel/otel-rn.adoc

```diff
@@ -12,9 +12,89 @@ You can use the {OTELName} xref:otel-forwarding-telemetry-data.adoc#otel-forward
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
-// git rebase marker 1
+[id="otel_3-4_{context}"]
+== Release notes for {OTELName} 3.4
 
-// git rebase marker 2
+The {OTELName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} <x>.<y>.<z>].
+
+The {OTELName} 3.4 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.<y>.0.
+
+[id="otel_3-4_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+
+[id="otel_3-4_technology-preview-features_{context}"]
+=== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+* OTLP JSON File Receiver
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="otel_3-4_new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* With this update, the following link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features reach General Availability:
+** BearerTokenAuth Extension
+** Kubernetes Attributes Processor
+** Spanmetrics Connector
+* ???.
```

Comment:
Multi-container auto-instrumentation injection for SDK

https://issues.redhat.com/browse/TRACING-4181 cc) @iblancasa

Reply:
Added. Thank you.